src/core/structured-log.js

```diff
@@ -277,12 +277,15 @@
       invoke('TRACE', mt, arguments);
     };
 
-    self.information = function(messageTemplate) {
+    self.verbose = self.trace;
```

Comment:
As part of this PR I have replaced 'trace' with 'verbose'.

I removed the 'debug' function. I'll bring up the addition of an official debug logging level in a separate issue.